packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -155,22 +156,42 @@ async function get_bundle(uid, mode, cache, lookup) {
 				// fetch from unpkg
 				self.postMessage({ type: 'status', uid, message: `resolving ${importee}` });
 
-				if (importer in lookup) {
-					const match = /^(@[^/]+\/)?[^/]+/.exec(importee);
-					if (match) imports.add(match[0]);
-				}
+				const importee_package_name_match = /^(@[^/]+\/)?[^/]+/.exec(importee);
+				const importee_package_name = importee_package_name_match
+					? importee_package_name_match[0]
+					: null;
 
-				try {
-					const pkg_url = await follow_redirects(`${packagesUrl}/${importee}/package.json`, uid);
-					const pkg_json = (await fetch_if_uncached(pkg_url, uid)).body;
-					const pkg = JSON.parse(pkg_json);
+				if (importee_package_name != null) {
+					if (importer in lookup) {
+						imports.add(importee_package_name);
+					}
 
-					if (pkg.svelte || pkg.module || pkg.main) {
-						const url = pkg_url.replace(/\/package\.json$/, '');
-						return new URL(pkg.svelte || pkg.module || pkg.main, `${url}/`).href;
+					try {
+						const pkg_url = await follow_redirects(
+							`${packagesUrl}/${importee_package_name}/package.json`,
+							uid
+						);
+						const pkg_json = (await fetch_if_uncached(pkg_url, uid)).body;
+						const pkg = JSON.parse(pkg_json);
+
+						/** @type {string | false | undefined} */
+						const resolved_id =
+							exprots_resolver(pkg, importee, { browser: true, conditions: ['production'] }) ??
+							legacy_resolver(pkg, {
+								browser: importee,
+								fields: [['svelte', 'browser', 'module', 'main']]
```

Comment:
I don't think this should be an array of arrays looking at the examples on https://github.com/lukeed/resolve.exports
```suggestion
								fields: ['svelte', 'browser', 'module', 'main']
```

Reply:
Woops! Totally right